### PR TITLE
Install guide refactor with pip dotnet

### PR DESCRIPTION
**Pull request by natke**
The builds on the changes in #527 but uses pip/dotnet instead of conda

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/TOC.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/19cfb498a4cc9cbb9cf4e3df40296b1cd1abaa44/articles/TOC.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6e215d3d-4cec-4bac-9737-549e4c3279de_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/19cfb498a4cc9cbb9cf4e3df40296b1cd1abaa44/articles/install-guide/index.md) | 95 | 90 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fa3316ec-3f44-4d4c-a62e-3874a5062b11_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

File: articles/install-guide/index.md [hunk 1]
```diff
@@ -1,53 +1,223 @@
...
+    - Create a Jupyter notebook with a Q# kernel, and add the following code to the first notebook cell:
+
+        ```qsharp
+        operation SayHello () : Unit {
+            Message("Hello from quantum world!");
+        }
+        ```
+
+    - Run this cell of the notebook:
+
```
Comment (cgranade) [inline articles/install-guide/index.md]: It looks like the actual cell is missing here?

image

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/TOC.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/362c687db4381ac51fb4a85c5d144a83d0a56d74/articles/TOC.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5ca6cbeb-5b68-4721-ae6e-998d7c9c01ea_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/362c687db4381ac51fb4a85c5d144a83d0a56d74/articles/install-guide/index.md) | 95 | 90 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1d6e5f1f-d96d-4b2a-a2ac-9a5157441a8c_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

File: articles/install-guide/index.md (hunk 1 above)
Comment (natke) [reply articles/install-guide/index.md]: I didn't intend the cell be there but I agree it is not clear. I've added a screenshot

**Pull request merged**